### ISSUE TITLE
NVDA not reading all menu items with Classic Shell in Windows 8

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2878]
**Reported by hurrikenny on 2012-12-21 05:48**
I have recently upgraded to Windows 8 pro. I didn't really like using the metro apps start menu, so I downloaded Classic Shell - which gave me back my traditional start menu on my desktop. The only issue I am having is that when I press the Windows key, and arrow up to the apps menus, and then say arrow right to one of them, it will read it then, but not when I arrow back to the left. If I then arrow up or down, it will read it. It does the same when I go to programs and a few other menus where I have to arrow back to the left. It reads fine using arrow up or arrow down, just not using the left arrow to go back. I am also aware that the search field is not labelled (which I will contact the developer to fix from their end). If I turn off NVDA and test it with Narrator, it reads all of the menus correctly. Here is a link to the program. http://download.cnet.com/Classic-Shell/3000-2072_4-75553853.html

**Comment by hatserdevlats** [comment]
Is it possible to add code for the Classic Shell start menu to the appmodule for Explorer?

**Comment by josephsl** [comment]
Hi, if Control+NVDA+F1 says “explorer.exe” while focused on Classic Shell, then yes (I’m thinking not). Thanks.

**Comment by bhavyashah** [comment]
Confirmed. The search edit field is read as 'edit blank' by NVDA, indicating that it is unlabelled from the software vendor's end itself. The second issue is that arrowing up to the Programs submenu, pressing the right arrow, and then pressing the left arrow does not take the user back to the Programs submenu but to some completely different place (PC username icon), something that seems less of an NVDA bug to me and more of a Classic Shell one. Regardless, I probably think the App Specific and NeedsExternalFix label would be suitable for this ticket.

**Comment by Adriani90** [comment]
@hurrikenny, @ghost is this still reproducible for you?

**Comment by Adriani90** [comment]
We didn't get any updates from the authors that could reproduce this issue for a very long time, and Support for Windows 8 ended completely in 2016 while Windows 8.1 ended completely in January 2023. I don't expect any futher updates for these Windows versions. I am closing this issue as abandoned given the missing updates.

**Comment by Brian1Gaff** [comment]
It was I think a classic shell issue. This software was very good in Windows 
7, I use it there now, but I could never get a lot of reliability for any 
Windows 8 installation.
 The software has now become legacy, there being a new one that covers 10 
and 11, but to me they are not so attractive as they have no facility to 
make Explorer ribbon-less as the previous version did on 7 and was supposed 
to do on 8.
 Brian

-- 
***@***.***
Sent via blueyonder.(Virgin media)
Please address personal E-mail to:-
***@***.***, putting 'Brian Gaff'
in the display name field.
----- Original Message -----
